js/layout-javascript/news-feed-code.js

```diff
@@ -165,6 +167,40 @@ DynamicList.prototype.attachObservers = function() {
         console.error(error);
       });
     })
+    .on('click', '.sort-group .list-sort li', function (e) {
+      e.stopPropagation();
+
+      var $sortListItem = $(e.currentTarget);
+      var oldSortOrder = $sortListItem.data('sortOrder');
+      var $sortOrderIcon = $sortListItem.find('i');
+      var $sortList = _this.$container.find('.list-sort li');
+      var sortClasses = {
+        'none': 'fa-sort',
+        'asc': 'fa-sort-asc',
+        'desc': 'fa-sort-desc'
+      };
+      var newSortOrder = {
+        'none': 'asc',
+        'asc': 'desc',
+        'desc': 'asc'
+      };
+
+      _this.sortOrder = newSortOrder[oldSortOrder];
+      _this.sortField = $sortListItem.data('sortField');;
+      _this.Utils.DOM.resetSortIcons({ $sortList: $sortList });
+
+      $sortOrderIcon.removeClass(_.values(sortClasses).join(' ')).addClass(sortClasses[_this.sortOrder]);
+      $sortListItem.data('sortOrder', _this.sortOrder);
+
+      _this.Utils.Records.sortByField({
+        $container: _this.$container,
+        listContainer: '#news-feed-wrapper-' + _this.data.id,
```

Comment:
We already have a `options.$container` being passed. Why do we need to pass this selector string just for it to be used to create a new `$(options.listContainer)` object?

Reply:
@tonytlwu the `listContainer` we are using to cut the items wrapper, so we could sort items in it and past it to the same place where it was.

Reply:
I see. Let's switch it up a little bit.

```suggestion
        $listContainer: $('#news-feed-wrapper-' + _this.data.id),
```